libs/Onboarding/src/Onboarding/Keychain/ServiceInterface.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <QtCore>
```

Comment:
I don't think you need all of `QtCore` in a header file; `std::weak_ptr` otoh is missing :)